docs/build_process.md

```diff
@@ -29,9 +29,6 @@ This document mentions several components that communicate with each other:
    1. The `BuildConfig`'s source git ref names the git branch the build is from
       ― The `bump_release` configuration names the branch's commit the initial
       build is from
-   1. The `stop_autorebuild_if_disabled` configuration names the config file
```

Comment:
Should this be in a separate commit?

Reply:
not sure if it is worth, just obsolete plugin in doc